pyaerocom/colocation/colocation_setup.py

```diff
@@ -464,12 +464,16 @@ def validate_no_forbidden_keys(self):
             if key in self.model_fields:
                 raise ValidationError
 
-    @cached_property
-    def basedir_logfiles(self):
-        p = Path(self.basedir_coldata) / "logfiles"
-        if not p.exists():
-            p.mkdir(parents=True, exist_ok=True)
-        return str(p)
+        return self
+
+    @model_validator(mode="after")
+    # @classmethod
+    def validate_start_stop_xand(self):
+        if not (self.start and self.stop):
+            if self.start or self.stop:
```

Comment:
are the nested `if` needed?
maybe this could be re-written as
``` python
        if not (self.start and self.stop) and self.start or self.stop:
```
or 
``` python
        if bool(self.start) != bool(self.stop):
```